lib/rucio/tests/test_api_external_representation.py

```diff
@@ -226,27 +231,16 @@ def test_api_replica(self):
         """ REPLICA (API): Test external representation of replicas """
 
         did = 'ext_' + str(generate_uuid())
-        add_replicas('MOCK', [{'scope': self.scope_name, 'name': did, 'bytes': 100}], issuer='root', **self.vo)
+        add_replicas(self.rse_name, [{'scope': self.scope_name, 'name': did, 'bytes': 100}], issuer='root', **self.vo)
 
         add_did(self.scope_name, 'ext_parent_2', 'dataset', issuer='root', account=self.account_name, **self.vo)
         attachment = {'scope': self.scope_name, 'name': 'ext_parent_2',
                       'dids': [{'scope': self.scope_name, 'name': did}]}
         attach_dids_to_dids([attachment], issuer='root', **self.vo)
 
-        # get_did_from_pfns - in progress but requires fix to stfc/rucio#33
```

Comment:
Do we not need to do this test anymore? It looks like it would check that we have internal scopes based on the `@` symbol, so I guess the fact we use external representations now means this isn't relevant?

Reply:
The replica is still returned with an internal scope.  The main reason I got rid of this is that I couldn't get it to work.  I might change the API to use `api_update_return_dict`, but also I just realized that I could potentially copy working parts from test_replica.py, so I'll try that too.